sdk/src/sdk/pubsub/MqttPubSubClient.cpp

```diff
@@ -21,6 +21,7 @@
 
 #include "sdk/middleware/Middleware.h"
 
+#include "mqtt/connect_options.h"
 #include <mqtt/async_client.h>
```

Comment:
Project-external includes shall use angle brackets:
```suggestion
#include <mqtt/async_client.h>
#include <mqtt/connect_options.h>
```